back/src/domain/core/eventBus/events.ts

```diff
@@ -110,6 +111,10 @@ export type DomainEvent =
 
 export type DomainTopic = DomainEvent["topic"];
 
+export type AssessmentEmailDomainTopic =
+  | "EmailWithLinkToCreateAssessmentSent"
+  | "BeneficiaryAssessmentEmailSent";
```

Comment:
un petit truc qui me gène, c'est que rien ici n'empèche de faire une faute, ou de mettre une string quelconque. Il faudrait s'assurer que on a bien un DomainTopic quand même. 

On peut faire un truc comme ça :
```suggestion
export type AssessmentEmailDomainTopic = ExtractFromExisting<
  DomainTopic,
  | "EmailWithLinkToCreateAssessmentSent"
  | "BeneficiaryAssessmentEmailSent"
  >
```